forte/data/caster.py

```diff
@@ -78,3 +81,46 @@ def input_pack_type():
     @staticmethod
     def output_pack_type():
         return MultiPack
+
+
+class DataPackBoxer(Caster[MultiPack, DataPack]):
+    """
+    This class creates a DataPack from a MultiPack, this DataPack is the only
+    content of the original MultiPack.
```

Comment:
Not sure if `this DataPack is the only content` is clear. It seems to imply the original MultiPack only contains one data pack.  Also, we don't "create" a DataPack here. In fact, what a caster does is to pass on the pack, here, instead of passing the multipack, it pass on a single data pack within this pack.